src/components/share-structure/ShareStructure.vue

```diff
@@ -1187,24 +1187,30 @@ export default class ShareStructure extends Vue {
   opacity: .4;
 }
 
+.v-btn {
+  text-transform: none !important;
+}
```

Comment:
Why do you need this? Please add a code comment.

Reply:
Are you saying all VButtons now display their label in uppercase? 🤦 

This will affect ALL our buttons. Instead of overriding all of them individually, I think we should add the override in the global assets folder (in override file). Thoughts?

Reply:
Shaoyun, what are you thoughts on this?

Reply:
> Are you saying all VButtons now display their label in uppercase? 🤦
> 
> This will affect ALL our buttons. Instead of overriding all of them individually, I think we should add the override in the global assets folder (in override file). Thoughts?

sorry for missing this messagekkk. looking it now!